ISISComputingGroup/EPICS-motorExtensions
# Ticket 2138: GEM beamscrapper jaws

Adds the GEM beamscraper jaws (https://github.com/ISISComputingGroup/IBEX/issues/2138). 

Ensure that you have the PRs that link to this ticket. 

To test:
* Confirm that the documentation at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/GEM-Beamscraper-Jaws makes sense, is mathematically correct and corresponds to the code
* Run the new IOC tests for the beamscraper jaws
* Switch to the NDXGEM config branch, modify the linmot to be in devsim
* Confirm that you can move the beamscraper jaws as expected
